Classes/Client.php

```diff
@@ -99,8 +99,16 @@ public function createGlossary(
 
         $formatEntries = '';
         foreach ($entries as $entry) {
-            $source = $entry['source'];
-            $target = $entry['target'];
+            /*
+             * as the version without trimming in TCA is already published,
+             * we trim a second time here
+             * to avoid errors in DeepL client
+             */
+            $source = trim($entry['source']);
```

Comment:
@calien666 @NarkNiro Not sure and I guess it may be safe. However, I would harden this here with:

1. `??` null coalscing fallback
2. add a `(string)` type case

Better be protected against `undefined array key access` and `non-string` values ?

Note: for `$source` and `$target`

Reply:
This behaviour is hardened by static code analyzer and therefor not needed